portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
+#background {
+    
+}
+
+#whoami {
+
+}
+
+#mybackground {
+
+}
+
+#interests {
+
+}
+
+#funfacts {
+
+}
+
+#contact {
+
+}
+
+#bar1, #bar2, #bar3 {
+  width: 35px;
```

Comment:
please keep CSS declarations in alphabetic order, per our style guide:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order